src/PlexRequests/Controllers/AuthController.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using PlexRequests.Core;
+using PlexRequests.Helpers;
+using PlexRequests.Models;
+using PlexRequests.Plex;
+using PlexRequests.Settings;
+using PlexRequests.Store.Models;
+
+namespace PlexRequests.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    public class AuthController : Controller
+    {
+        private readonly IUserService _userService;
+        private readonly IPlexApi _plexApi;
+        private readonly AuthenticationSettings _authSettings;
+        private readonly ILogger<AuthController> _logger;
+
+        public AuthController(
+            IUserService userService,
+            IPlexApi plexApi,
+            IOptions<AuthenticationSettings> authSettings,
+            ILogger<AuthController> logger)
+        {
+            _userService = userService;
+            _plexApi = plexApi;
+            _authSettings = authSettings.Value;
+            _logger = logger;
+        }
+
+        [HttpPost]
+        [Route("Login")]
+        [AllowAnonymous]
+        public async Task<IActionResult> Login(UserLoginRequest request)
+        {
+            _logger.LogDebug("Attempting Plex SignIn");
+
+            var plexUser = await _plexApi.SignIn(request.Username, request.Password);
+
+            if (plexUser == null)
+            {
+                _logger.LogDebug("Invalid PlexCredentials");
+                return BadRequest("Unable to login to Plex with the given credentials");
```

Comment:
You returning a plain string without a json format? Is this as designed or out of laziness?

Reply:
Yep. Did this before i put in exception handling and obviously forgot to convert it over.